frontends/web/src/routes/exchange/pocket.tsx

```diff
@@ -212,6 +212,7 @@ export const Pocket = ({ code, action }: TProps) => {
       sendAll: 'no',
       selectedUTXOs: [],
       paymentRequest: message.slip24,
+      useHighestFee: true
```

Comment:
Shouldn't setting useHighestFee be enough or  it still required/makes-sense  to pass: `feeTarget: 'custom', customFee: '',`?

Reply:
It's still a required field but ignored if `useHighestFee` is true, just like previously when there was a payment request